src/date.rs

```diff
@@ -0,0 +1,61 @@
+/// wasm32-unknown-unknown polyfill
+use cfg_if::cfg_if;
+
+cfg_if! {
```

Comment:
You don't need to use `cfg_if`, just do:

```rust
#[cfg(target_arch = "wasm32", target_os = "unknown")]
mod js_sys_data {
    // <js_sys code here>
}

#[cfg(target_arch = "wasm32", target_os = "unknown")]
pub use js_sys_data::OffsetDateTime;
#[cfg(not(target_arch = "wasm32", target_os = "unknown"))]
pub use time::OffsetDateTime;
```

I'd like to do this instead to keep the dependencies to an absolute minimum. The use-case is simple enough to not warrant an external dependency.

Reply:
Ok, will remove the dep of cfg_if.